tests/build.proj

```diff
@@ -54,7 +54,7 @@
 
   <Target Name="RestorePackage">
     <PropertyGroup>
-      <DotnetCliPath Condition="'$(DotnetCliPath)'==''">$(ProjectDir)..\.dotnet\</DotnetCliPath>
+      <DotnetCliPath Condition="'$(DotnetCliPath)'==''">$(RepoRoot)\.dotnet\</DotnetCliPath>
```

Comment:
I cannot find where the RepoRoot comes from.

Reply:
This gets set internally in the Arcade scripts.

Reply:
Ok, I see. I had a two weeks old local checkout which didn't contain that yet.